features/kafka/consumer/src/main/proto/kafka-consumer-events.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+option java_package = "org.opennms.features.kafka.consumer.events";
+option java_outer_classname = "EventsProto";
+
+// The values differ from the standard codes in OpenNMS
+// since proto3 enforces us to start at 0
```

Comment:
Oof... I don't suppose we could just make 0 be "unknown" or something, and have it not be used? kinda sucks to have them off-by-one.

Reply:
Why not do the industry standard, use `UNUSED`, `PAD0`, `PAD1`, or `RESERVED`?

Reply:
`INDETERMINATE` seems to be 0 so there is already `unknown` case.  Mapping seems to be easier by matching label.

Reply:
@Naicisum I think this issue is mostly trying to use our `Event` and match it with protobuf and get events consumed directly on OpenNMS.

Reply:
Sure, but `INDETERMINATE` has a specific meaning/mapping in the OpenNMS code, whereas in this case the 0th entry would be not actually used.  I am not a fan of having 2 different "standards" for what the enums map top (0-indexed and 1-indexed), it makes it the API ambiguous and have different behavior whether you're parsing say, the ReST API vs the Kafka stream.

Reply:
This was copied from here [opennms-kafka-producer.proto#L7](https://github.com/OpenNMS/opennms/blob/opennms-28.0.0-2/features/kafka/producer/src/main/proto/opennms-kafka-producer.proto#L7) for reference - so we already to the start at 0 thing in the producer case.

Reply:
oof, alright... then I guess there are [2 standards](https://xkcd.com/927/)  :D